src/surface_conditions/surface_conditions.jl

```diff
@@ -26,9 +26,19 @@ function update_surface_conditions!(Y, p, t)
         Fields.field_values(Fields.level(Fields.coordinate_field(Y.c).z, 1))
     sfc_conditions_values = Fields.field_values(sfc_conditions)
     wrapped_sfc_setup = sfc_setup_wrapper(sfc_setup)
-    sfc_temp_var =
-        p.atmos.surface_model isa PrognosticSurfaceTemperature ?
-        Fields.field_values(Y.sfc.T) : nothing
+    if p.atmos.sfc_temperature isa ExternalTVColumnSST
+        evaluate!(
+            p.external_forcing.surface_inputs.ts,
+            p.external_forcing.surface_timevaryinginputs.ts,
+            t,
+        )
+        sfc_temp_var = Fields.field2array(p.external_forcing.surface_inputs.ts)
```

Comment:
I think this is incorrect and it should be
```suggestion
        sfc_temp_var = Fields.field_values(p.external_forcing.surface_inputs.ts)
```

Reply:
@Sbozzolo Agreed ! 

@Julians42 : `field2array` returns an array of size `Nv x Ncols` (e.g. radiation internals) whereas we want to get the appropriate datalayout with the appropriate # of indices for 2d planes / columns. 

```
e.g.  # The exact field below is just one such demonstration case I generate in a REPL session (in a global run) : 

julia> Fields.field_values(test_field) |> typeof
ClimaCore.DataLayouts.VIJFH{ClimaCore.Geometry.WVector{Float32}, 11, 4, Array{Float32, 5}}

julia> Fields.field2array(test_field) |> typeof
Matrix{Float32} (alias for Array{Float32, 2})

julia> Fields.field2array(test_field) |> size
(11, 3456)

julia> Fields.field_values(test_field) |> size
(4, 4, 1, 11, 216)
```